From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: celenity <celenity@celenity.dev>
Date: Sun, 05 Jul 2026 20:19:20 +0000
Subject: [PATCH] Gecko - Ensure builds always reflect our (latest) changes

Ensures that builds of GeckoView binaries always reflect our (latest) changes.

Signed-off-by: celenity <celenity@celenity.dev>
---
diff --git a/mobile/android/gradle/with_gecko_binaries.gradle b/mobile/android/gradle/with_gecko_binaries.gradle
index 7998bdc894..fd4c477bd3 100644
--- a/mobile/android/gradle/with_gecko_binaries.gradle
+++ b/mobile/android/gradle/with_gecko_binaries.gradle
@@ -20,7 +20,7 @@ ext.configureVariantWithGeckoBinaries = { variantName ->
     // the native binaries.  To do this, the Gradle build calls out to the
     // moz.build system, which can be re-entrant.  Official builds are driven by
     // the moz.build system and should never be re-entrant in this way.
-    if (hasCompileArtifacts() && !mozconfig.substs.MOZILLA_OFFICIAL && !mozconfig.substs.ENABLE_MOZSEARCH_PLUGIN) {
+    if (hasCompileArtifacts() && !mozconfig.substs.ENABLE_MOZSEARCH_PLUGIN) {
         // `onVariants` runs before AGP creates these tasks, so defer the wiring with `configureEach`.
         ["generate${variantName.capitalize()}Assets", "merge${variantName.capitalize()}JniLibFolders"].each { taskName ->
             tasks.matching { it.name == taskName }.configureEach {
-- 
